cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -913,6 +914,18 @@ function Configure-CniNetworking {
   Log-Output "CNI config:`n$(Get-Content -Raw ${l2bridge_conf})"
 }
 
+# Obtain the host dns conf and save it to a file so that kubelet/CNI can use it to configure
+# dns suffix search list for pods
+function Configure-HostDNSConf {
+  $server_ip = Get-DnsClientServerAddress -InterfaceAlias "vEthernet (Ethernet)" | Select-Object -ExpandProperty ServerAddresses
+  $search_list = Get-DnsClient | Select-Object -ExpandProperty ConnectionSpecificSuffixSearchList
```

Comment:
Fyi, in other places in this file I think we access properties by wrapping the command in (), then using the `.` operator. I'm not sure if that's the same thing as `Select-Object -ExpandProperty`...

Reply:
Updated. Thanks for your review!